src/typechecker/FStar.TypeChecker.TcTerm.fs

```diff
@@ -1084,11 +1125,14 @@ and tc_value env (e:term) : term
   | Tm_uinst({n=Tm_fvar fv}, us) ->
     let us = List.map (tc_universe env) us in
     let (us', t), range = Env.lookup_lid env fv.fv_name.v in
+    maybe_warn_on_use env fv;
```

Comment:
Great to see this at typechecking time, but somehow the warning doesn't trigger for tactics:
```fstar
let cheat : squash False =    
    _ by (let lid = pack_fv ["Prims"; "admit"] in    
          let t : term = Tv_FVar lid in    
          exact (`((`#t) ())))    
```
I'll look and let you know, probably something weird in tactics.

Reply:
Oh, this actually happens because of `env.lax || env.phase1` check in `maybe_warn_on_use`--- neither is true for tactics. The easy fix might be to keep a `.tactic` flag?

Reply:
actually, why not just replace this check for `not env.phase1`?

Reply:
ah, thanks. I can do that.

Reply:
I'm removing this guard altogether, since from the other comment, we will report only unique errors. So, let's log the issue on every use, whether lax, phase1, phase2, whatnot. And leave it to FStar.Errors to decide how to report it